### PR TITLE
Adding new image for ci-operator with ubi base image

### PR DESCRIPTION
**Pull request by deepsm007**
Adding a new image for smooth transition from centos:stream9 to ubi-minimal

/cc @danilo-gemoli @openshift/test-platform

### CODE REVIEW

File: images/ci-operator-minimal/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM registry.access.redhat.com/ubi9/ubi-minimal:latest
+
+RUN microdnf install -y git python3
```
Comment (droslean) [inline images/ci-operator-minimal/Dockerfile:3]: Can you check if we can deprecate that? IIRC the git dependency exists to allow users to pass their own git revisions to test stuff. I thing we long passed this idea. Also, python3, why?

Comment (danilo-gemoli) [reply images/ci-operator-minimal/Dockerfile:3]: I can't tell anything about git but python is required, see [this](https://redhat-internal.slack.com/archives/CBN38N3MW/p1717595757867709)

**Pull request merged**